guides/source/debugging_rails_applications.md

```diff
@@ -344,6 +344,8 @@ Processing by PostsController#index as HTML
 (rdbg)
 ```
 
+You can leave a debugging session with `quit` or `q` and confirm your action at the prompt or with `quit!` and its alias `q!` to bypass the prompt. In most cases though, you'd want to continue program execution with `continue` or `c`.
```

Comment:
I agree that in most cases the user will want to continue, so let's mention that first:

```suggestion
You can exit the debugging session at any time and continue your application execution with the `continue` (or `c`) command.  Or, to exit both the debugging session and your application, use the `quit` (or `q`) command.
```

I added "at any time" because we've just introduced the debugging session, and we don't want to exit it just yet, based on the next section of the guide.  (I also left out the `quit!` / `q!` command for brevity.)

Reply:
This definitely better. Thanks ;)